### PR TITLE
Sheldon branch

### PR DESCRIPTION
**Pull request by sdownie2**

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="" vcs="Git" />
+  </component>
+</project>
```
Comment (bantinglu) [inline .idea/vcs.xml]: what are these .idea files? do we need them? if not can you add them to .gitignore?

File: AndroidGesture/app/src/main/java/com/example/ben/androidgesture/GestureUtil.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        // return whatever direction was detected
+        Gesture gesture = new Gesture(Direction.UP);
+        return gesture;
+    }
+
+    public static void load(){
+        // loads new test data?
+    }
+
+}
```
Comment (bantinglu) [inline AndroidGesture/app/src/main/java/com/example/ben/androidgesture/GestureUtil.java]: Can you make a utils folder and move this class and all associated classed into it?

File: AndroidGesture/app/src/main/java/com/example/ben/androidgesture/PebbleReceiver.java [hunk 3]
```diff
@@ -21,6 +24,7 @@
 public class PebbleReceiver extends Activity {
 
     private static final String TAG = "PebblePointer";
+    private static final String DIRECTORY_PATH = Environment.getExternalStorageDirectory().toString();
```
Comment (bantinglu) [inline AndroidGesture/app/src/main/java/com/example/ben/androidgesture/PebbleReceiver.java:27]: I just realized we need a constants folder for these. I will do it.

File: AndroidGesture/app/src/main/java/com/example/ben/androidgesture/PebbleReceiver.java [hunk 4]
```diff
@@ -137,4 +143,24 @@ public void onPause() {
         PebbleKit.closeAppOnPebble(getApplicationContext(), PEBBLE_UUID);
     }
 
+    public void saveDataToFile(String x, String y, String z){
```
Comment (bantinglu) [inline AndroidGesture/app/src/main/java/com/example/ben/androidgesture/PebbleReceiver.java]: finalize the parameters

File: AndroidGesture/app/src/main/java/com/example/ben/androidgesture/PebbleReceiver.java [hunk 5]
```diff
@@ -137,4 +143,24 @@ public void onPause() {
...
+        try {
+            Timestamp timestamp = new Timestamp(System.currentTimeMillis());
+            File root = new File(DIRECTORY_PATH);
+            File file = new File(root, "logFile.txt");
+            if(!file.exists()){
+                file.createNewFile();
+            }
+            FileOutputStream fOut = new FileOutputStream(file);
+            OutputStreamWriter outWriter = new OutputStreamWriter(fOut);
+            outWriter.append(x + " " + y + " " + z + " " + timestamp.toString());
```
Comment (bantinglu) [inline AndroidGesture/app/src/main/java/com/example/ben/androidgesture/PebbleReceiver.java]: can you add the strings 'X: ' and etc to this line? I want to be able to know which ones the x, y, z without just memorizing the order

File: src/c/PebbleAccelerometer.c [hunk 6]
```diff
@@ -1,12 +1,11 @@
 #include <pebble.h>
-
+#define SOME_NUMBER 10
```
Comment (bantinglu) [inline src/c/PebbleAccelerometer.c]: remove.

**Comment by sdownie2** [comment]
did my changes go through that I made in response to your comments?

File: AndroidGesture/app/src/main/java/com/example/ben/androidgesture/gestureutil/Direction.java [hunk 7]
```diff
@@ -0,0 +1,9 @@
+package com.example.ben.androidgesture.gestureutil;
```
Comment (bantinglu) [inline AndroidGesture/app/src/main/java/com/example/ben/androidgesture/gestureutil/Direction.java:1]: I'd prefer to rename the file to just 'Utils', that can be done later though

File: AndroidGesture/app/src/main/java/com/example/ben/androidgesture/gestureutil/Gesture.java [hunk 8]
```diff
@@ -0,0 +1,19 @@
+package com.example.ben.androidgesture.gestureutil;
+
+
+/**
+ * Created by Sheldon on 2/1/2017.
+ */
+
+public final class Gesture {
+
+    Direction direction;
```
Comment (bantinglu) [inline AndroidGesture/app/src/main/java/com/example/ben/androidgesture/gestureutil/Gesture.java:10]: this could be final too, if we really wanna be safe.

**Review by bantinglu — APPROVED** [review]

**Pull request merged**